.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,25 @@
+---
+name: Feature request
+about: Suggest an idea for a11yphant
+title: ''
+labels: enhancement
+assignees: ''
+
+---
+
+Feature requests are very welcome. But take a moment to find out whether your idea fits with the scope and aims of the project. It's up to you to make a strong case to convince us of this feature. Please provide as much detail and context as possible.
+
+**Please discuss your proposed changes with us before starting implementation. Otherwise, they might collide with our plans.**
```

Comment:
Should we remove this here too? Or leave it as a warning not to implement anything?